app/controllers/contacts.js

```diff
@@ -0,0 +1,93 @@
+import BaseController from "./base/base.js";
+import Contact from "./../models/contact.js";
+import User from "./../models/user.js";
+import { ERROR_STATUES } from "../validations/constants/errors.js";
+
+class ContactsController extends BaseController {
+  constructor() {
+    super();
+  }
+
+  async contact_add(ws, data) {
+    const {
+      id: requestId,
+      contact_add: { email, phone },
+    } = data;
+
+    if (email) {
+      for (let i = 0; i < email.length; i++) {
+        const value = email[i].value;
+        const findedUser = await User.findOne({ email: value });
```

Comment:
I believe we can do a single request to DB for all emails,  for a sake of performance